### PR TITLE
B-19723-INT Office viewers can view EMPLID

### PR DESCRIPTION
**Pull request by paulstonebraker**
## Summary

Part of the effort to incorporate EMPLID into the app for USCG customers. Office users should now see the EMPLID in the header for office users, as well as the Move Details Customer Info section, if the customer is USCG

### How to test

1. Create a new customer, branch USCG and enter an emplid
2. Submit a move
3. Login to the office application and find the move - confirm that all places where the DODID is displayed, the EMPLID is as well
4. Confirm that non USCG customers do not have EMPLID displayed anywhere

### CODE REVIEW

File: src/components/Office/DefinitionLists/CustomerInfoList.jsx [hunk 1]
```diff
@@ -20,6 +21,12 @@ const CustomerInfoList = ({ customerInfo }) => {
           <dt>DoD ID</dt>
           <dd data-testid="dodId">{customerInfo.dodId}</dd>
         </div>
+        {customerInfo.agency === departmentIndicators.COAST_GUARD && (
+          <div className={descriptionListStyles.row}>
+            <dt>EMPLID</dt>
+            <dd data-testid="dodId">{customerInfo.emplid}</dd>
```
Comment (pambecker) [inline src/components/Office/DefinitionLists/CustomerInfoList.jsx:27]: the testid name needs update to emplid

**Comment by pambecker** [comment]
I am logged in as a SC viewing PPM closeouts and clicked on the Download AOA Paperwork link and still see the DoDID for a Coast Guard member.
image

File: src/components/Office/DefinitionLists/CustomerInfoList.jsx (hunk 1 above)
Comment (paulstonebraker) [reply src/components/Office/DefinitionLists/CustomerInfoList.jsx]: change pushed! Good eye

**Comment by pambecker** [comment]
> I am logged in as a SC viewing PPM closeouts and clicked on the Download AOA Paperwork link and still see the DoDID for a Coast Guard member. image

This work has been moved to another backlog.  Good to go

**Review by pambecker — APPROVED** [review]

**Review by loganwc — APPROVED** [review]

**Pull request merged**